engine/apps/api/views/integration_heartbeat.py

```diff
@@ -20,10 +20,17 @@ class IntegrationHeartBeatView(
     viewsets.GenericViewSet,
 ):
     authentication_classes = (PluginAuthentication,)
-    permission_classes = (IsAuthenticated, ActionPermission)
-    action_permissions = {
-        IsAdmin: (*MODIFY_ACTIONS, "activate", "deactivate"),
-        AnyRole: (*READ_ACTIONS, "timeout_options"),
+    permission_classes = (IsAuthenticated, RBACPermission)
+    rbac_permissions = {
+        # TODO: what permissions should go here?
+        "list": [],
```

Comment:
I assume this should be `RBACPermission.Permissions.INTEGRATIONS_<READ|WRITE>` here?

Reply:
should be `Integrations`